src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -44,16 +44,16 @@ class SeekBar extends Slider {
     // Avoid mutating the prototype's `children` array by creating a copy
     options.children = [...options.children];
 
-    const shouldDisableSeekWhileScrubbingOnMobile = player.options_.disableSeekWhileScrubbingOnMobile && (IS_IOS || IS_ANDROID);
+    const shouldDisableSeekWhileScrubbing = (player.options_.disableSeekWhileScrubbingOnMobile && (IS_IOS || IS_ANDROID)) || (player.options_.disableSeekWhileScrubbingOnSTV);
```

Comment:
This line feels very long, is the linter catching this? Either way, I'd probably break this into multiple lines of code
```suggestion
    const shouldDisableSeekWhileScrubbing =
      (player.options_.disableSeekWhileScrubbingOnMobile && (IS_IOS || IS_ANDROID))
      || (player.options_.disableSeekWhileScrubbingOnSTV);
```

Or something along those lines :)